### PR TITLE
Configured Dependabot 

### LINKED ISSUE #451: Set up a Dependabot
## Problem
Handling dependency upgrades is cumbersome.

## Description
Configure a dependabot to monitor upgrades and suggest them via automated PR's.

## Additional context
* https://github.com/creativecommons/vue-vocabulary/issues/230
* https://github.com/creativecommons/vue-vocabulary/pull/232
* https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/

 Update Propagation from vue-vocabulary as suggested by @dhruvkb.

## Implementation

- [x] I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by nimishbongale**
## Fixes
Fixes #451 by @nimishbongale and @dhruvkb 

## Description
Adds a `.github/dependabot.yml` file to configure dependabot.
The confguration was successful in my fork 

* https://github.com/nimishbongale/vocabulary/runs/763009948

## Screenshots

dependabot

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
```
Comment (dhruvkb) [inline .github/dependabot.yml]: Our YML files ([exhibit A](https://github.com/creativecommons/vocabulary/blob/master/docker-compose.yml) and [exhibit B](https://github.com/creativecommons/vocabulary/blob/master/.cc-metadata.yml)) have used single-quotes in their strings.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+version: 2
```
Comment (dhruvkb) [inline .github/dependabot.yml:6]: Leave a line after version. Looks cleaner imo.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
It's good to merge barring except for a couple of minor changes. Do you think comments would help?

**Comment by nimishbongale** [comment]
> It's good to merge barring except for a couple of minor changes. Do you think comments would help?

@dhruvkb Yeah, although I did feel the .yml file was pretty self-explanatory. I'd be fine either way; surely adding comments makes the file more reader friendly.

File: .github/dependabot.yml (hunk 2 above)
Comment (nimishbongale) [reply .github/dependabot.yml:6]: Alright!

File: .github/dependabot.yml (hunk 1 above)
Comment (nimishbongale) [reply .github/dependabot.yml]: Right. Will change!

File: .github/dependabot.yml (hunk 2 above)
Comment (nimishbongale) [reply .github/dependabot.yml:6]: Done.

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+#################################
+# Dependabot Configuration File #
+#################################
+
+# This is the current version of the github native dependabot
```
Comment (dhruvkb) [inline .github/dependabot.yml]:
```suggestion
# This is the current version of the GitHub-native Dependabot
```

**Review by dhruvkb — APPROVED** [review]

**Pull request merged**